Modules/TPC/src/CheckOfTrendings.cxx

```diff
@@ -128,197 +132,227 @@ Quality CheckOfTrendings::check(std::map<std::string, std::shared_ptr<MonitorObj
   if (!canv) {
     ILOG(Fatal, Support) << "Canvas not found" << ENDM;
   }
-  TGraph* g = getGraph(canv);
-  if (!g) { // if there is no TGraph, give an error and break
-    ILOG(Fatal, Support) << "No TGraph found to perform Check against." << ENDM;
-  }
-
-  const int nBins = g->GetN();
-  const double* yValues = g->GetY();
-  const double* yErrors = g->GetEY(); // returns nullptr for TGraph (no errors)
+  getGraphs(canv, graphs);
 
-  bool useErrors = true;
-  if (yErrors == nullptr) {
-    useErrors = false;
-    ILOG(Info, Support) << "NO ERRORS" << ENDM;
+  if (graphs.size() == 0) {
+    ILOG(Fatal, Support) << "Could not retrieve any TGraph for CheckOfTrendings" << ENDM;
+  }
+  for (int iGraph = 0; iGraph < graphs.size(); iGraph++) {
+    if (!graphs[iGraph]) { // if there is no TGraph, give an error and break
+      ILOG(Fatal, Support) << "TGraph number " << iGraph << " is NULL." << ENDM;
+    }
+  }
+  if (!mSliceTrend && graphs.size() > 1) {
+    ILOG(Fatal, Support) << "Multiple Graphs found even though this is not a slice trending" << ENDM;
   }
 
-  double mean = 0.;
-  double stddevOfMean = 0.;
+  for (int iGraph = 0; iGraph < graphs.size(); iGraph++) {
```

Comment:
This will give a warning, since int is signed and `....size()` unsigned (`size_t`). I propose to avoid to get the warning and use either
```suggestion
  for (size_t iGraph = 0; iGraph < graphs.size(); iGraph++) {
```
Or, if you know the number is smaller than the max value of int
```suggestion
  for (int iGraph = 0; iGraph < std::static_cast<int>(graphs.size()); iGraph++) {
```
Please change it accordingly in all places.

Reply:
Done